packages/app/app/components/EqualizerPresetList/index.tsx

```diff
@@ -1,12 +1,17 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import { List, Icon } from 'semantic-ui-react';
 import classNames from 'classnames';
 import { useTranslation } from 'react-i18next';
 
 import styles from './styles.scss';
 
-const EqualizerPresetList = ({ presets, onClickItem, selected }) => {
+type EqualizerPresetListProps ={
+  presets: Array<any>;
```

Comment:
Please don't use any

Reply:
Removed the use of Any from both of the components